api/src/main/java/com/pip/chatbot/service/JokeService.java

```diff
@@ -23,4 +22,8 @@ public JokeService(JokeDao jokeDao) {
     public List<Joke> getAll(){
         return jokeDao.getAll();
     }
+
```

Comment:
Tak jak w Dao zmieńmy typ zwracany na Optional<Joke>. Tak nam poleci null na pustą bazę, a tego nie chcemy. Lepiej dać jawnie to zrozumienia że czegoś może nie być:)